src/almanack/metrics/remote.py

```diff
@@ -38,6 +39,14 @@ def get_api_data(
     """
     if params is None:
         params = {}
+    
+    github_token = os.environ.get("GITHUB_TOKEN")
+    headers = {"accept": "application/json"}
+    if github_token and ("github.com" in api_endpoint or "api.github.com" in api_endpoint):
+        headers["Authorization"] = f"Bearer {github_token}"
+    else:
+        headers = {"accept": "application/json"}
```

Comment:
Nice! Consider adding a short comment above this block to help describe why / when this might be needed.